{{cookiecutter.root_dir__update_if_you_intend_to_use_monorepo}}/README.md

```diff
@@ -25,10 +25,27 @@ After that, follow the [ML pull request guide](docs/ml-pull-request.md)
-| Data Scientist                | Update production ML code (e.g. model training logic) for an existing project | [ML pull request guide](docs/ml-pull-request.md)                                                                                                                     |
-| Data Scientist                | Modify production model ML resources, e.g. model training or inference jobs   | [ML resource config guide]({{cookiecutter.project_name}}/terraform/README.md)                                                                                        |
-| MLOps / DevOps                | Set up CI/CD and ML pipeline resource deployment for the current ML project   | [MLOps setup guide](docs/mlops-setup.md)                                                                                                                             |
+| Role                          | Goal                                                                          | Docs                                                                                                                                                                |
+|-------------------------------|-------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| First-time users of this repo | Understand the ML pipeline and code structure in this repo                    | [Project overview](docs/project-overview.md)                                                                                                                        |
+| Data Scientist                | Get started writing ML code for a brand new project                           | {% if cookiecutter.include_feature_store == "yes" %}[ML quickstart](docs/ml-developer-guide-fs.md).{% else %}[ML quickstart](docs/ml-developer-guide.md){% endif %} |
+| Data Scientist                | Update production ML code (e.g. model training logic) for an existing project | [ML pull request guide](docs/ml-pull-request.md)                                                                                                                    |
+| Data Scientist                | Modify production model ML resources, e.g. model training or inference jobs   | [ML resource config guide]({{cookiecutter.project_name}}/terraform/README.md)                                                                                       |
+| MLOps / DevOps                | Set up CI/CD and ML pipeline resource deployment for the current ML project   | [MLOps setup guide](docs/mlops-setup.md)                                                                                                                            |
+
+## Mono repo
```

Comment:
```suggestion
## Monorepo
```

Reply:
Thank you. Updated all `Mono repo` => `Monorepo` and `poly repo` => `polyrepo.`